jina/__init__.py

```diff
@@ -42,7 +44,7 @@ def _warning_on_one_line(message, category, filename, lineno, *args, **kwargs):
 
     _set_start_method(_start_method.lower())
     _warnings.warn(f'multiprocessing start method is set to `{_start_method.lower()}`')
-    _os.unsetenv('JINA_MP_START_METHOD')
+    _os.environ.pop('JINA_MP_START_METHOD')
```

Comment:
but is this the same as `unsetenv`, isn't `dict.pop` modify the dict object itself instead of really unset env vars?

Reply:
This seems to do the same thing as `unsetenv` (tested on Ubuntu, python 3.7.11)

```python
import os
import subprocess

echo = lambda x, y: print(x, '\t', subprocess.check_output([f'echo ${y}'], shell=True, universal_newlines=True).strip())

def _pop():
    var_name = 'FOO_POP'
    os.environ[var_name] = 'BAR'
    echo('after set env: ', var_name)
    os.environ.pop(var_name)
    echo('after os.environ.pop: ', var_name)

def _unsetenv():
    var_name = 'FOO_UNSET'
    os.environ[var_name] = 'BAR'
    echo('after set env: ', var_name)
    os.unsetenv(var_name)
    echo('after os.unsetenv: ', var_name)

def _del():
    var_name = 'FOO_DEL'
    os.environ[var_name] = 'BAR'
    echo('after set env: ', var_name)
    del os.environ[var_name]
    echo('after del os.environ: ', var_name)

_pop()
_unsetenv()
_del()
```

```
after set env:           BAR
after os.environ.pop:    
after set env:           BAR
after os.unsetenv:       
after set env:           BAR
after del os.environ: 
```

Reply:
interesting! thanks for explaining it